.github/CODEOWNERS

```diff
@@ -2,6 +2,22 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners
 # This will automatically assign a team / people as reviewers for PRs based on the files changed within the PR.
 
+# Core Workflows team's main responsibilities include policies including default policies and policy management workflows,
+# detection and alerting, integrations and notifiers, risk, vulnerability management and reporting.
+pkg/detection/**/*                  @stackrox/core-workflows
+pkg/booleanpolicy/**/*              @stackrox/core-workflows
+pkg/networkgraph/**/*               @stackrox/core-workflows
+pkg/defaults/policies/**/*          @stackrox/core-workflows
+proto/**/*                          @stackrox/core-workflows
+central/vulnerabilityrequest/**/*   @stackrox/core-workflows
+central/reports/**/*                @stackrox/core-workflows
+central/reportconfiguration/**/*    @stackrox/core-workflows
+central/networkgraph/**/*           @stackrox/core-workflows
+central/networkpolicies/**/*        @stackrox/core-workflows
+central/networkentities/**/*        @stackrox/core-workflows
+sensor/**/*                         @stackrox/core-workflows
```

Comment:
Best practice for CODEOWNERS is to really only put a barriers up if the team _must_ review it. That's usually because it's dangerous (think auth), risky to touch or because it might require other unknown changes (dependencies etc). It's not intended as an FYI thing because this is a blocking operation. 

In other words by putting all of proto on here we're basically signing up to do all reviews _and_ blocking other teams even if it doesn't impact us at all. 

So with that I don't think it makes sense to put it. I really don't want to review any change to sensor for e.g. even if it has nothing to do with our area of ownership.

If you feel strongly on the notifier proto (for example), then I would scope this down to just those sets of protos.

Reply:
Makes sense let me start with a trimmed down version with policies detection and vuln

Reply:
Removed, added specific protos of interest instead